README.md

```diff
@@ -32,7 +32,7 @@ To do so, in your project `build.sbt` add a dependency on
 ```
 There are a number of different ways to specify this dependency in the build.sbt file. If you have based your circuit on the 
 [Chisel-template](https://github.com/freechipsproject/chisel-template.git) the addition should look like
-```scala
+```
```

Comment:
No idea why, I'll probably just put it back

Reply:
I'm putting it back, I removed it because IntelliJ was complaining that the scala code in there was broken, but I don't think that's enough reason to change it.